packages/frontend-v2/components/Header/AccountOverview.tsx

```diff
@@ -52,8 +52,8 @@ export const AccountOverview: React.FC<AccountOverviewProps> = ({
 
   useEffect(() => {
     reset({
-      firstName: student.fullName.split(" ")[0],
-      lastName: student.fullName.split(" ")[1],
+      firstName: student.fullName?.split(" ")[0],
+      lastName: student.fullName?.split(" ")[1],
```

Comment:
Are we saying that fullnames are optional here or may not exist?

Reply:
may not exist because they are optional!